src/actions/incidentAction.jsx

```diff
@@ -17,7 +13,7 @@ export const loadIncidentsSuccess = incidents => {
 export const loadIncidents = () => {
   return dispatch => {
     return axios
-      .get(url(), {})
+      .get(config.INCIDENTS_URL, {})
```

Comment:
You don't need the empty  `{}` if you are not providing any options.

Reply:
Noted